### PR TITLE
CAT-STRING string update

### PR DESCRIPTION
**Pull request by jonny911**

### CODE REVIEW

**Comment by thmq** [comment]
please fix checkstyle issues (i.e. remove the unused import).

File: catroid/src/main/res/values-ar/strings.xml [hunk 1]
```diff
@@ -258,18 +316,18 @@
...
+        The source code of Pocket Code is mainly licensed under the GNU AGPL v3 License.
         For further details on the license see the link below.\n</string>
   <string name="dialog_about_pocketcode_license_link_text">رخصة مصدر النص البرمجي لجهاز الجيب</string>
   <string name="dialog_about_catrobat_link_text">حول كاتروبات</string>
   <!-- Project activity view -->
   <string name="project_name">البرنامج:</string>
   <string name="my_projects_activity_title">البرامج</string>
   <string name="number_of_objects">Objects:</string>
-  <string name="start_scene_name">\"%1$s\" (First Scene)</string>
+  <string name="start_scene_name">"%1$s" (First Scene)</string>
```
Comment (robertpainsi) [inline catroid/src/main/res/values-ar/strings.xml]: I'm not sure if this is on purpose. Quotes that aren't escaped, will be stripped.
https://jira.catrob.at/browse/CAT-2147

File: catroid/src/main/java/org/catrobat/catroid/ui/fragment/ScenesListFragment.java [hunk 2]
```diff
@@ -396,7 +396,8 @@ public void onCancel(DialogInterface dialogInterface) {
 	private void checkSceneCountAfterDeletion() {
 		ProjectManager projectManager = ProjectManager.getInstance();
 		if (projectManager.getCurrentProject().getSceneList().size() == 0) {
-			Scene emptyScene = new Scene(getActivity(), String.format(getString(R.string.default_scene_name), 1), projectManager.getCurrentProject());
+			Scene emptyScene = new Scene(getActivity(), getString(R.string.default_scene_name) + " 1", projectManager
+					.getCurrentProject());
```
Comment (robertpainsi) [inline catroid/src/main/java/org/catrobat/catroid/ui/fragment/ScenesListFragment.java]: For internationalization, strings displayed to the user shouldn't be concatenated.
English: `Scene 1`
Arabic: `مشهد 1` (translated by google)

Comment (robertpainsi) [inline catroid/src/main/java/org/catrobat/catroid/ui/fragment/ScenesListFragment.java]: :+1: Good catch!

I wrote an email to @AyyalAwwad also addressing name enumeration yesterday. Really looking forward to his feedback. Unfortunately it's not as easy as a simple string translation with format specifiers.

**Comment by thmq** [comment]
Thank you @jonny911 ❤️

**Pull request merged**